# generated by patch-package 6.4.7 on 2021-08-29 19:40:28
#
# command:
#   npx patch-package mdsvex
#
# declared package:
#   mdsvex: https://registry.npmjs.org/mdsvex/-/mdsvex-0.9.8.tgz
#
diff --git a/node_modules/mdsvex/dist/browser-es.js b/node_modules/mdsvex/dist/browser-es.js
index 6bc648d..de20604 100644
--- a/node_modules/mdsvex/dist/browser-es.js
+++ b/node_modules/mdsvex/dist/browser-es.js
@@ -31499,6 +31499,10 @@ function parse$7(value, settings) {
 
         namedEntity = type === name$1 ? decodeEntity_browser(characters) : false;
 
+        // QUICKFIX https://github.com/pngwn/MDsveX/issues/296
+        if (namedEntity == '<') namedEntity = '&lt;';
+        if (namedEntity == '>') namedEntity = '&gt;';
+
         if (namedEntity) {
           entityCharacters = characters;
           entity = namedEntity;
diff --git a/node_modules/mdsvex/dist/browser-umd.js b/node_modules/mdsvex/dist/browser-umd.js
index a838231..ccf260b 100644
--- a/node_modules/mdsvex/dist/browser-umd.js
+++ b/node_modules/mdsvex/dist/browser-umd.js
@@ -31505,6 +31505,10 @@
 
           namedEntity = type === name$1 ? decodeEntity_browser(characters) : false;
 
+          // QUICKFIX https://github.com/pngwn/MDsveX/issues/296
+          if (namedEntity == '<') namedEntity = '&lt;';
+          if (namedEntity == '>') namedEntity = '&gt;';
+
           if (namedEntity) {
             entityCharacters = characters;
             entity = namedEntity;
diff --git a/node_modules/mdsvex/dist/main.cjs.js b/node_modules/mdsvex/dist/main.cjs.js
index c4f1c09..be098c3 100644
--- a/node_modules/mdsvex/dist/main.cjs.js
+++ b/node_modules/mdsvex/dist/main.cjs.js
@@ -8719,6 +8719,10 @@ function parse$7(value, settings) {
 
         namedEntity = type === name$1 ? decodeEntity_1(characters) : false;
 
+        // QUICKFIX https://github.com/pngwn/MDsveX/issues/296
+        if (namedEntity == '<') namedEntity = '&lt;';
+        if (namedEntity == '>') namedEntity = '&gt;';
+
         if (namedEntity) {
           entityCharacters = characters;
           entity = namedEntity;
diff --git a/node_modules/mdsvex/dist/main.es.js b/node_modules/mdsvex/dist/main.es.js
index 779eb25..30f8ccc 100644
--- a/node_modules/mdsvex/dist/main.es.js
+++ b/node_modules/mdsvex/dist/main.es.js
@@ -8689,6 +8689,10 @@ function parse$7(value, settings) {
 
         namedEntity = type === name$1 ? decodeEntity_1(characters) : false;
 
+        // QUICKFIX https://github.com/pngwn/MDsveX/issues/296
+        if (namedEntity == '<') namedEntity = '&lt;';
+        if (namedEntity == '>') namedEntity = '&gt;';
+
         if (namedEntity) {
           entityCharacters = characters;
           entity = namedEntity;
